fe/styles.css

```diff
@@ -0,0 +1,192 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Anton', sans-serif;
+}
+
+body {
+    padding-top: 60px;
+}
+
+header {
+    background-image: linear-gradient(
+            rgba(0, 0, 0, 0.2),
+            rgba(0, 0, 0, 0.2)
+    ), url('../images/recordsHero.jpg');
+    width: 80%;
+    height: 300px;
+    text-align: center;
+    margin: 0 auto 4% auto;
+}
+
+header h1 {
+    color: RGB(255,255,255);
+    bottom: 50%;
+    font-size: 5em;
+    margin: 0;
+    line-height: 300px;
+    letter-spacing: 0.05em;
+}
+
+main {
+    width: 80%;
+    margin: 0 auto;
+}
+
+.surround {
+    overflow: auto;
+    background-image: linear-gradient(
+            rgba(0, 0, 0, 0.2),
+            rgba(0, 0, 0, 0.2)
+    ), url('https://media.giphy.com/media/yY3KurO43569O/giphy.gif');
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: auto 370px;
+    margin-bottom: 60px;
+}
+
+.floater {
+    width: 50%;
+    height: 100%;
+    float: left;
+    text-align: center;
+}
+
+img {
+    display: block;
+    border: 5px solid;
+    margin-left:0;
+    margin-right: auto;
+    height: 330px;
+    width: 330px;
+}
+
+#info {
+    height: 330px;
+    text-align: center;
+    font-size: 1.1em;
+}
+
+#info ul {
+    background-image: url('../images/dataBack.jpeg');
+    list-style: none;
+    color: RGB(255,255,255);
+    padding: 6% 0 0 0;
+    border: 5px solid black;
+    width: 330px;
+    margin: 0;
+    height: 330px;
+    line-height: 300%;
+    margin-left: auto;
+    margin-right: 0;
+}
+
+a {
+    text-decoration: none;
+    color: RGB(255,255,255);
+}
+
+.floater a {
+    background-color: RGBA(0,0,0,0);
+    color: RGB(255,255,255);
+    font-size: 1.2em;
+    padding: 2px 6px 2px 6px;
+    border: 5px solid RGB(255,255,255);
+}
+
+footer a{
+    display: block;
+    margin-top: 20px;
+}
+
+.addRecord {
+    display: inline-block;
+    background-color: RGBA(0,0,0,0);
+    color: RGB(255,255,255);
+    font-size: 1.2em;
+    padding: 2px 6px 2px 6px;
+    border: 5px solid RGB(255,255,255);
+}
+
+footer {
+    background-image: linear-gradient(
+            rgba(0, 0, 0, 0.2),
+            rgba(0, 0, 0, 0.2)
+    ), url('../images/recordsHero.jpg');
+    width: 80%;
+    height: 150px;
+    text-align: center;
+    margin: 60px auto;
+}
+
+@media only screen and (max-width: 900px) {
+    .surround{
+        background: RGB(255,255,255);
+        background-size: 330px;
+    }
+}
+
+@media only screen and (max-width: 850px) {
+    img {
+        display: block;
```

Comment:
here the only thing that changes anything in the browser is the width definition.